Library/Formula/llvm.rb

```diff
@@ -118,6 +118,8 @@ def install
       args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
     end
 
+    args << "-DCMAKE_CXX_FLAGS=#{ENV['CPPFLAGS']}" unless OS.mac?
```

Comment:
CMake doesn't seem to accept environmental CPPFLAGS which, in case of Homebrew, contain include paths. CXXFLAGS were set correctly from what I remember.

Lack of this line resulted in compilation errors with histedit.h not found.